### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by jahinzee** [LGUG2Z/helix-vim#23]
The README states the following, which isn't very clear.:

> Feel free to fork this, personalize it, make suggestions, improvements, fill in gaps that I don't know how to fill, etc.

I would appreciate it if you would license this with a proper license, and indicating it through LICENSE file. GitHub has tools to make adding a license easy to do.

**Comment by 3ynm** [comment]
I don't think this project _requires_ a licence, but probably @LGUG2Z would do some users a favour by making his intentions clear by placing or linking the [CC0](https://creativecommons.org/publicdomain/zero/1.0/) license.
